SocialFilter/sections/css_extended.txt

```diff
@@ -2,6 +2,7 @@
 ! Extended CSS rules
 !
 1zaicev.ru#?#article > .text-center:has(> p:contains(Поделитесь статьёй))
+gran-turismo.com#?#div:matches-css(float: right) > a:matches-css(margin-left: 12px):matches-css(cursor: pointer):not([href])
```

Comment:
better try to block these buttons it by class name, and check after few days
```
gran-turismo.com##.kLGGEq > a[class="_3cOCYi _3FunNS"]
gran-turismo.com##.kLGGEq > a[class="_3cOCYi gwHPsB"]
```

Reply:
I exchanged for now to simple rules. Also added mobile-app banner.